website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -2,25 +2,23 @@ This component takes advantage of the `sort-by` helper provided in [ember-compos
 
 ## How to use this component
 
-### Non-sortable table
-
-If you don’t have or want to use a model, a basic invocation could look like:
+### Non-sortable table with no model defined, caption defined
```

Comment:
I am not sure we should qualify normal tables as "non-sortable". They're just tables.
Plus, I would remove the "caption defined" from the title, for simplicity (if you want to stress the importance of adding a caption argument, I would add a specific example in the `More examples" or have a dedicated section about it in the "Accessibility" panel ("Best practices").

```suggestion
### Table with no model defined
```